config/locales/de.helpers.yml

```diff
@@ -18,6 +18,9 @@ de:
       new: "Neu"
       edit: "Bearbeiten"
       show: "Anzeigen"
+      assign_ownership: "Als Captain hinzufügen"
+      delete_ownership: "Als Captain löschen"
+      delete_membership: "Löschen"
```

Comment:
Analogous to the English localization I would prefer something like “Aus dem Team entfernen” to clarify that not the user himself but only the membership is going to be deleted.